liquibase-core/src/main/java/liquibase/diff/output/changelog/AbstractChangeGenerator.java

```diff
@@ -136,22 +136,24 @@ private String doctor(String stringToDoctor) {
     }
 
     private void setPropertyIgnoreSchemaAndCatalogCase(Change change, String field, String value, CompareControl.SchemaComparison comparison) {
-        if (comparison.getOutputSchemaAs() != null
-                && comparison.getComparisonSchema() != null
+        if ("catalogName".equalsIgnoreCase(field)) {
```

Comment:
The idiom used in the rest of the file looks like this:
```
field.toLowerCase().contains("catalogname")
```
Just for consistency, we should use the same syntax.

Reply:
Honestly I didn't catch an idea so far :-) 
but let it be 
I've changed to 
```
     if (field != null && field.toLowerCase().contains("catalogname")) {
            return;
        }
```